enhanced/metrics.go

```diff
@@ -365,51 +404,57 @@ var Metrics = map[string]Metric{
 			[]string{"instance", "region"},
 			map[string]string(nil),
 		),
+		Unit: 1,
 	},
 	"node_memory_Active": {
 		Name: "active",
 		Desc: prometheus.NewDesc(
 			"node_memory_Active",
-			"The amount of assigned memory, in kilobytes.",
+			"The amount of assigned memory, in bytes.",
 			[]string{"instance", "region"},
 			map[string]string(nil),
 		),
+		Unit: 1024,
 	},
 	"node_memory_Buffers": {
 		Name: "buffers",
 		Desc: prometheus.NewDesc(
 			"node_memory_Buffers",
-			"The amount of memory used for buffering I/O requests prior to writing to the storage device, in kilobytes.",
+			"The amount of memory used for buffering I/O requests prior to writing to the storage device, in bytes.",
 			[]string{"instance", "region"},
 			map[string]string(nil),
 		),
+		Unit: 1024,
 	},
 	"node_memory_Cached": {
 		Name: "cached",
 		Desc: prometheus.NewDesc(
 			"node_memory_Cached",
-			"The amount of memory used for caching file system–based I/O.",
+			"The amount of memory used for caching file system–based I/O, in bytes.",
 			[]string{"instance", "region"},
 			map[string]string(nil),
 		),
+		Unit: 1024,
 	},
 	"node_memory_nr_dirty": {
```

Comment:
hm, I am not sure, but maybe conversion is not needed here.
please check

Reply:
`node_memory_nr_dirty` is delivered from `memory->dirty` metric and it has description:
> The amount of memory pages in RAM that have been modified but not written to their related data block in storage, in kilobytes. 

(source: https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_Monitoring.OS.html)

Reply:
@rnovikovP @arvenil yes, we have kilobytes in AWS.
I think we have kilobytes also in node_exporter for these metrics...
it means - maybe conversion is not needed

Reply:
node_exporter gives metrics node_vmstat_pswpin/out in pages.
So values are multiplied by page size (4096) in formulas.

Reply:
node_memory_  metrics are collected in bytes by node_exporter